src/foam/comics/v2/DAOBrowseControllerView.js

```diff
@@ -0,0 +1,112 @@
+/**
+ * @license
+ * Copyright 2019 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.comics.v2',
+  name: 'DAOBrowseControllerView',
+  extends: 'foam.u2.View',
```

Comment:
Let's add documentation: '' to these classes just to briefly describe what they do.

Reply:
Done.